docs/contributor/03-11-hap-rules.md

```diff
@@ -4,11 +4,11 @@
 > This feature is still being developed and will be available soon.
 
 The Hyperscaler Account Pool (HAP) rule configuration is a functionality that allows you to control pool selection from a configurable helm property. 
-A pool is a set of Gardener's SecretBinding resources that share the same search labels. 
-Pool selection is a process that decides which SecretBinding pool is the most suitable for every SAP BTP, Kyma runtime creation request.
-The process generates a list of [search labels](#search-labels) based on Kyma runtime's attributes.
+A pool is a set of Gardener's SecretBinding custom resources that share the same labels. 
+Pool selection is a process that decides which Secret Binding pool is the most suitable for every SAP BTP, Kyma runtime creation request.
+The process generates a list of requirements for a [label selector](#Label-Selector) based on Kyma runtime's attributes.
```

Comment:
```suggestion
The process generates a list of requirements for a [label selector](#label-selector) based on Kyma runtime's attributes.
```

Reply:
The link checker reports the link as broken. Let's see if it is still so after removing the capital letters.

Reply:
Suggestion applied.